### DISCUSSION TITLE
Inquiry about Conformance Class 8: 'INSPIRE data sets and data set series linked service metadata'

### DISCUSSION THREAD

**Discussion by AljosaCekic** [INSPIRE-MIF/helpdesk-validator#1087] [Q&A] [answered]
Hello,

here is Aljosa Cekic from the  State Office for Geoinformation and Land Development Baden-Württemberg (LGL), the Competence Center for Geodata Infrastructure.

I would have a question regarding the calculation of INSPIRE-Indicators. Could you answer me whether the check of Conformance Class 8: 'INSPIRE data sets and data set series linked service metadata' for the metadata of data sets and data set series is obligatory for the INSPIRE-Indicators calculation? 

I am looking forward for your answer.

Kind regards,
Aljosa Cekic

**Comment by MarcoMinghini** [comment, marked as answer]
Dear @AljosaCekic, 
for the calculation of the INSPIRE Monitoring indicator on the conformity of dataset metadata (indicator MDi1.1), Conformace Class 8 is not used. This is because Conformance Class 8 implements the [good practice on data-service linking simplification](https://github.com/INSPIRE-MIF/gp-data-service-linking-simplification), which - although being a recommended and more efficient approach than the traditional one - is only an alternative way to encode dataset metadata.
Best regards.

**Reply by AljosaCekic** [reply]
Dear @MarcoMinghini,

thank you very much for your response and for the clarification.

Kind regards,
Aljosa Cekic
